### DISCUSSION TITLE
when i run curl in powershell .

### DISCUSSION THREAD

**Discussion by Shaiks123** [hyperledger/fabric#3555] [Q&A]
/bin/bash: line 1: $'\r': command not found
/bin/bash: line 2: ?xml: No such file or directory
/bin/bash: line 3: !DOCTYPE: No such file or directory
/bin/bash: line 4: http://www.w3.org/TR/xhtml1/DTD/xhtml1-strict.dtd: No such file or directory
/bin/bash: line 5: html: No such file or directory
/bin/bash: line 6: head: No such file or directory
/bin/bash: line 7: title: No such file or directory
/bin/bash: line 8: /head: No such file or directory
/bin/bash: line 9: body: No such file or directory
/bin/bash: line 10: h1: No such file or directory
/bin/bash: line 11: p: No such file or directory
/bin/bash: line 12: h3: No such file or directory
/bin/bash: line 13: p: No such file or directory
/bin/bash: line 14: hr: No such file or directory
/bin/bash: line 15: p: No such file or directory
/bin/bash: line 16: /body: No such file or directory
/bin/bash: line 17: /html: No such file or directory

**Comment by bestbeforetoday** [comment]
The only help I can offer is that, when running or developing against Fabric on my Windows machine, I do everything with an Ubuntu image in Windows Subsystem for Linux 2 (WSL2). That works fine for me. I use VS Code with the Remote Development extension (or IntelliJ for Java) on my desktop to work with code inside the WSL2 image.
